Unic.Bob.Scratch.nuspec

```diff
@@ -1,16 +1,18 @@
 <?xml version="1.0"?>
-  <package >
+  <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
     <metadata>
       <id>Unic.Bob.Scratch</id>
       <authors>Unic AG</authors>
       <version>$VERSION$</version>
       <owners>Unic AG</owners>
-      <iconUrl>https://www.unic.com/img/unic-logo.png</iconUrl>
+      <iconUrl>https://github.com/unic/unic/raw/master/unic_nuget.png</iconUrl>
+      <projectUrl>https://github.com/unic/bob-scratch</projectUrl>
       <requireLicenseAcceptance>false</requireLicenseAcceptance>
       <description>Scratch install Sitecore with SIF.</description>
     </metadata>
     <files>
+        <file src="Installation.config" />
```

Comment:
if we manage to move/remove installation.config, this entry is not required anymore.